satellite/console/service.go

```diff
@@ -55,37 +56,36 @@ const (
 
 // Service is handling accounts related logic
 type Service struct {
-	Signer
+	log *zap.Logger
 
-	store DB
-	log   *zap.Logger
+	Signer
+	store         DB
+	stripeService payments.Service
 
 	passwordCost int
 }
 
 // NewService returns new instance of Service
-func NewService(log *zap.Logger, signer Signer, store DB, passwordCost int) (*Service, error) {
+func NewService(log *zap.Logger, signer Signer, store DB, stripeService payments.Service, passwordCost int) (*Service, error) {
 	if signer == nil {
 		return nil, errs.New("signer can't be nil")
 	}
-
 	if store == nil {
 		return nil, errs.New("store can't be nil")
 	}
-
 	if log == nil {
 		return nil, errs.New("log can't be nil")
 	}
-
 	if passwordCost == 0 {
```

Comment:
I see that we are checking all the arguments against its zero value here. Do we need to also add the check for `stripeService`?

Reply:
I don't think we need it for internal pkg